test/run_tests.sh

```diff
@@ -108,6 +108,7 @@ function run_op_tests {
   run_dynamic python3 "$CDIR/../../test/nn/test_embedding.py" "$@" -v TestEmbeddingNNDeviceTypeXLA
   run_dynamic python3 "$CDIR/../../test/test_type_promotion.py" "$@" -v TestTypePromotionXLA
   run_dynamic python3 "$CDIR/test_operations.py" "$@" --verbosity=$VERBOSITY
+  run_dynamic python3 "$CDIR/test_dynamic_shapes.py" "$@"
```

Comment:
Don't you need to turn on experimental shapes

Reply:
`run_dynamic` will enable those flags. 
https://github.com/pytorch/xla/blob/515d53e9d78b5788e3f0b351b50b9b4423d0e80e/test/run_tests.sh#L64-L71